app/models/operation.rb

```diff
@@ -57,4 +57,37 @@ def reset_plan_id
       self.hrd_id = nil
     end
   end
+
+  def regional_request_in_operation
+    query = <<-SQL
```

Comment:
Here is an alternative way to get the same result using ActiveRecord:
RegionalRequest.joins(:operation).where(:operation_id => self.id).group(:region_id, :operation_id).count(:id)

Reply:
Neat! I'll change it to AR then.